authors/apps/articles/tests/test_ratings.py

```diff
@@ -0,0 +1,212 @@
+import copy
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+from rest_framework.reverse import reverse as api_reverse
+from rest_framework import status
+
+from authors import settings
+
+
+class RatingsTest(APITestCase):
+    """
+    Tests rating of an article
+    """
+    def setUp(self):
```

Comment:
Cool, will work on this.

Reply:
I found out I couldn't actually use the base methods as I need to create a user with custom attributes. This is because a user cannot rate their own article. With the overridden version though, I can create both users as required.

This also applies to `create_article` as with the overridden version, I can pass the token of the authenticated author.